contracts/Aavegotchi/ForgeDiamond/facets/ForgeTokenFacet.sol

```diff
@@ -114,7 +114,7 @@ contract ForgeTokenFacet is Modifiers {
         uint256 amount,
         bytes memory data
     ) public {
-        require(from == LibMeta.msgSender() || isApprovedForAll(from, LibMeta.msgSender()), "ForgeTokenFacet: caller is not token owner or approved");
```

Comment:
Why was this change necessary? @infiniRekt

Reply:
transfers were reverting when called from the vrf facet.